src/app/scheduling/scheduling-routes-data.ts

```diff
@@ -18,6 +18,8 @@ export const schedulingRoutesData = {
     "datepicker-sample-7": { displayName: "Dropdown Datepicker Template", parentName: "Datepicker" },
     "datepicker-sample-8": { displayName: "Custom Action Buttons", parentName: "Datepicker" },
     "datepicker-sample-9": { displayName: "Range Datepicker", parentName: "Datepicker" },
+    "datetimeeditor-sample-1": {displayName: "Date Time Editor Sample", parentName: "Date Time Editor"},
```

Comment:
Rename "Date Time Editor Sample" to the more meaningful "Date Time Basic" in all instances.